postman-generator/src/main/scala/generator/SetupCleanupFolderBuilder.scala

```diff
@@ -39,8 +39,8 @@ object SetupCleanupFolderBuilder {
         setupItem -> deleteItemOpt
     }
 
-    val setupSteps = setupItemToCleanupItemOpts.map(_._1)
-    val cleanupSteps = setupItemToCleanupItemOpts.flatMap(_._2).reverse
+    val setupSteps = setupItemToCleanupItemOpts.map(_._1).distinct
```

Comment:
these `distinct`s are redundant.
`DependantOperationResolver` does it on the attributes level